spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -183,30 +183,27 @@
     RUBY
   end
 
-  it 'uses relative path' do
-    allow(RuboCop::PathUtil)
-      .to receive(:relative_path).and_return('spec/models/bar_spec.rb')
+  it 'registers an offense for a path containing the class name' do
     expect_offense(<<-RUBY, '/home/foo/spec/models/bar_spec.rb')
       describe Foo do; end
       ^^^^^^^^^^^^ Spec path should end with `foo*_spec.rb`.
     RUBY
   end
 
-  it 'uses relative path for sibling directory project' do
-    allow(RuboCop::PathUtil)
-      .to receive(:relative_path)
-      .and_return('../ext-project/spec/models/bar_spec.rb')
-    expect_no_offenses(<<-RUBY, '/home/ext-project/spec/models/bar_spec.rb')
-      describe Bar do; end
+  it 'detects the path based on a class name with long module' do
+    expect_offense(<<-RUBY, '/home/foo/spec/very/my_class_spec.rb')
+      describe Very::Long::Namespace::MyClass do; end
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Spec path should end with `very/long/namespace/my_class*_spec.rb`.
     RUBY
   end
 
-  it 'uses relative path for different path project' do
-    allow(RuboCop::PathUtil)
-      .to receive(:relative_path)
-      .and_return('../../opt/ext-project/spec/models/bar_spec.rb')
-    expect_no_offenses(<<-RUBY, '/opt/ext-project/spec/models/bar_spec.rb')
-      describe Bar do; end
+  it 'detects the path based the absolute path of the file' do
+    allow(File).to receive(:expand_path).and_call_original
```

Comment:
Is this needed? If you've experienced an issue with RSpec going nuts about stubbed `File.expand_path`, it's been [fixed in RSpec 3.10](https://github.com/rspec/rspec-support/pull/431/commits/d18bd38f608eff053a32bcbc9d3604f280c8942f#diff-99389396657b227e4704a1f3baef7a701d143906dd68f114dbf1aa42c790c1beR15).

Reply:
If you'd like to update our used RSpec version and remove this like, please go ahead. Preferably in a separate pull request though. But I consider this optional.

Reply:
You are right. It's not needed. I guess I was running on older rspec version before.

Looking at my `Gemfile.lock`, it's already using rspec 3.10.0. Looking at the project gemspec, it should be already using the latest version.

I removed the line and tests seem to pass.